app/controllers/protocols_controller.rb

```diff
@@ -610,7 +610,7 @@ def protocolsio_import_create
       end
       return 0 # return 0 stops the rest of the controller code from executing
     end
-    if file_size / 1000 > Constants::FILE_MAX_SIZE_MB
+    if file_size > Constants::FILE_MAX_SIZE_MB * 1000000
```

Comment:
You can use the function `megabytes` like this ` Constants::FILE_MAX_SIZE_MB.megabytes`